SU-HSDO/suhumsci
# STN-689 : HOTFIX Pull Decanter Mappings

# [STN-689](https://sparkbox.atlassian.net/browse/STN-689) READY FOR REVIEW

## Summary
Pull the decanter class mappings implemented in #803 as they could have unintended consequences on legacy sites.

## Need Review By (Date)
asap

## Urgency
super high 🔥

## Steps to Test
1. In your CLI, run `npm run test` and confirm all tests pass.
2. Confirm the following classes have been removed from the work implemented in #803.
    - [x] decanter-caption
    - [x] decanter-credit
    - [x] decanter-font-splash
    - [x] decanter-font-lead
    - [x] decanter-button
    - [x] decanter-more-link
3. Confirm that the decanter classes above no longer receive Humsci Basic styles. You can use the copy and paste the code snippet below into a WYSIWGY text area to compare humsci basic styles with decanter styles. Decanter classes should be devoid of any styling.
</br>

```
<h2>Captions:</h2>
<p class="hs-caption">hs-caption</p>
<p class="decanter-caption">decanter-caption</p>
<p class="caption">caption</p>
<hr />
<h2>Credits:</h2>
<p class="hs-credits">hs-credits</p>
<p class="decanter-credit">decanter-credit</p>
<p class="credits">credits</p>
<hr />
<h2>Info Text:</h2>
<p class="hs-font-splash">hs-font-splash</p>
<p class="decanter-font-splash">decanter-font-splash</p>
<p class="infotext">infotext</p>
<hr />
<h2>Summary/Font lead:</h2>
<p class="hs-font-lead">hs-font-lead</p>
<p class="decanter-font-lead">decanter-font-lead</p>
<p class="summary">summary</p>
<hr />
<h2>Button:</h2>
<p class="hs-button"><a href="http://google.com">hs-button on the p tag</a></p>
<p class="decanter-button"><a href="http://google.com">decanter-button on the p tag</a></p>
<p class="btn"><a href="http://google.com">btn on the p tag</a></p>
<p>--</p>
<p><a class="hs-button" href="http://google.com">hs-button on the a tag</a></p>
<p><a class="decanter-button" href="http://google.com">decanter-button on the a tag</a></p>
<p><a class="btn" href="http://google.com">btn on the button tag</a></p>
<hr />
<h2>More Link:</h2>
<p class="hs-more-link"><a href="http://google.com">hs-more-link on the p tag</a></p>
<p class="decanter-more-link"><a href="http://google.com">decanter-more-link on the p tag</a></p>
<p>--</p>
<p><a class="hs-more-link" href="http://google.com">hs-more-link on the a tag</a></p>
<p><a class="decanter-more-link" href="http://google.com">decanter-more-link on the a tag</a></p>
```
